helm-chart/templates/cronjob.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.rolling_restart.enabled }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "json-rpc-relay.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  concurrencyPolicy: Forbid
+  schedule: '0 0 * * *'
```

Comment:
Can we go with `@daily` or `@midnight` to make this more readable

Reply:
For this I have variablized the `rolling_restart.schedule` and the default is set to `@daily` in values.yaml